### DISCUSSION TITLE
CSS gone after upgrade

### DISCUSSION THREAD

**Discussion by niels-przybilla** [postalserver/postal#1865] [Help with using Postal]
Hello,

I upgraded today postal with postal upgrade. It seems that everything went right. All systems seem to be running but GUI is broken.

<img width="699" alt="My Settings - Postal 2022-02-14 19-46-34" src="https://user-images.githubusercontent.com/99684821/153928507-789bc2ed-9400-4efd-a1b7-d9964191dddb.png">

How can I repair the GUI ?

THX and best wishes

Niels

**Comment by willpower232** [comment]
There is another change to make to your nginx configuration, have a look at https://github.com/postalserver/postal/issues/1602#issuecomment-926448573
